src/helpers/filters.ts

```diff
@@ -30,11 +30,11 @@ async function getFilters({
   ];
 
   let err: error = null;
-  if (event === EVENT_PULL_REQUEST) {
+  if (event === EVENT_PULL_REQUEST && !process.env.SVGO_ACTION_E2E_TEST) {
     const [f, err0] = await filters.NewPrFilesFilter({ client, context });
     result.push(f);
     err = err0;
-  } else if (event === EVENT_PUSH) {
+  } else if (event === EVENT_PUSH && !process.env.SVGO_ACTION_E2E_TEST) {
```

Comment:
This is a temporary fix that should be either replaced (i.e. solution outside of the code) or improve (i.e. solution at a higher level in the code). See also 23ead821f74ec59740f590a21c3e56fd79b4433d

Reply:
See #447